src/main/kotlin/uk/gov/justice/digital/hmpps/hmppstier/controller/DeliusCommunityDataReliability.kt

```diff
@@ -71,37 +71,34 @@ class DeliusCommunityDataReliability(
     ],
   )
   @PreAuthorize("hasRole('ROLE_HMPPS_TIER')")
-  @GetMapping("/crn/all")
-  suspend fun getAllDataReliability(): Flow<CommunityDeliusData> {
-    return tierReader.getCrns().mapNotNull {
+  @GetMapping("/crn/all/{offset}/{limit}")
+  suspend fun getAllDataReliability(@PathVariable(required = true) offset: Int, @PathVariable(required = true) limit: Int): Flow<CommunityDeliusData> {
+    return tierReader.getCrns(offset, limit).mapNotNull {
       val deliusInputs = try {
-        tierToDeliusApiService.getTierToDelius(it)
+        tierToDeliusApiClient.getDeliusTierTest(it)
       } catch (e: WebClientException) {
         log.error("Webclient exception in Tier To Delius for CRN: $it", e)
-        DeliusInputs(false, BigDecimal.valueOf(-1), -10, false, false, Registrations(false, emptyList(), null, null))
+        null
```

Comment:
how will you know if tier to delius returns a 404 and community API returns a 200 if you are filtering out all deliusInputs?

Reply:
yes, this will be a separate PR, at the moment I want to see how this will behave when deployed and if we can get rid of the time out errors. We will also need to add requirements.